src/mon/Monitor.cc

```diff
@@ -5875,7 +5875,8 @@ int Monitor::get_auth_request(
   bufferlist *out)
 {
   std::scoped_lock l(auth_lock);
-  if (con->get_peer_type() != CEPH_ENTITY_TYPE_MON) {
+  if (con->get_peer_type() != CEPH_ENTITY_TYPE_MON
+      && con->get_peer_type() != CEPH_ENTITY_TYPE_MGR) {
```

Comment:
even though this `&&` on the second line is driving me nuts :/

Reply:
fixed

Reply:
I guess I'll need to re-build the packages because of line 5887?

Reply:
Yes